### PR TITLE
Mage_Log - DOC block update

### PR DESCRIPTION
**Pull request by sreichel**
- doc blocks added/fixed
- PSR2 fixes (whitespaces, linebreaks, ...)

### CODE REVIEW

File: app/code/core/Mage/Log/Model/Log.php [hunk 1]
```diff
@@ -58,6 +58,9 @@ protected function _construct()
         $this->_init('log/log');
     }
 
+    /**
+     * @return float|int
```
Comment (tmotyl) [inline app/code/core/Mage/Log/Model/Log.php:62]: why float? the config description is "Save Expired Items Days" and default value is "180"

File: app/code/core/Mage/Log/Model/Visitor.php [hunk 2]
```diff
@@ -158,6 +187,9 @@ public function getUrl()
         return $url;
     }
 
+    /**
+     * @return mixed
```
Comment (tmotyl) [inline app/code/core/Mage/Log/Model/Visitor.php:191]: why mixed? I wold expect rather int

File: app/code/core/Mage/Log/Model/Visitor.php [hunk 3]
```diff
@@ -166,6 +198,9 @@ public function getFirstVisitAt()
         return $this->getData('first_visit_at');
     }
 
+    /**
+     * @return mixed
```
Comment (tmotyl) [inline app/code/core/Mage/Log/Model/Visitor.php:202]: same here

File: app/code/core/Mage/Log/Model/Log.php (hunk 1 above)
Comment (sreichel) [reply app/code/core/Mage/Log/Model/Log.php:62]: 'Mage::getStoreConfig(self::XML_LOG_CLEAN_DAYS)' returns a 'string' ... multiplied its float.

Comment (tmotyl) [reply app/code/core/Mage/Log/Model/Log.php:62]: ok, so we need a casting to int here.
I propose we either skip this docbloc change, and open a new PR fixing the docblock and adding cast.

File: app/code/core/Mage/Log/Model/Visitor.php (hunk 2 above)
Comment (sreichel) [reply app/code/core/Mage/Log/Model/Visitor.php:191]: I've used mixed only in a very few cases, where multiple return types are possble. Mixed/object are on todo list as mentioned in #784.

File: app/code/core/Mage/Log/Model/Visitor.php (hunk 3 above)
Comment (sreichel) [reply app/code/core/Mage/Log/Model/Visitor.php:202]: See above.

File: app/code/core/Mage/Log/Model/Log.php (hunk 1 above)
Comment (tmotyl) [reply app/code/core/Mage/Log/Model/Log.php:62]: created an issue for this change https://github.com/OpenMage/magento-lts/issues/949

**Comment by tmotyl** [comment]
thanks @sreichel ! :tada:

**Comment by sreichel** [comment]
> 
> 
> thanks @sreichel ! tada

I am happily surprised :)

Didnt check ... if this is discussed in discord plmk.

**Pull request merged**